README.md

```diff
@@ -1,30 +1,20 @@
-compliance
-==========
+# CTK
+An extensible Conformance Test Kit for the GA4GH Server
```

Comment:
You are correct.  There's been some recent discussion about creating client libraries, each of which would of course have an API, so I'll rephrase this as

```
An extensible Conformance Test Kit for the GA4GH Server API.
```
How does that strike you?